pkg/url/url.go

```diff
@@ -191,11 +283,40 @@ func List(client *occlient.Client, localConfig *config.LocalConfigInfo, componen
 	return urlList, nil
 }
 
-func getProtocol(route routev1.Route) string {
-	if route.Spec.TLS != nil {
-		return "https"
+// ListIngress returns all Ingress URLs for given component.
+func ListIngress(client *kclient.Client, envSpecificInfo *envinfo.EnvSpecificInfo, componentName string) (iextensionsv1.IngressList, error) {
```

Comment:
Why are there two list functions with different names? They seem to be doing the same thing and envSpecificInfo is unused in this function.

Reply:
I was planning to use `ListIngress` to list all local ingresses and pushed ingresses, also add state to each ingress found. 
Then found out it might won't be a small change, need to rethink and rewrite some code. 
Delete the `ListIngress` for now. Added a `TODO` comment
The work for it will be in https://github.com/openshift/odo/issues/2787

Reply:
OK :+1: